lighthouse-core/audits/accessibility/td-headers-attr.js

```diff
@@ -26,7 +26,7 @@ const UIStrings = {
   description: 'Screen readers have features to make navigating tables easier. Ensuring ' +
       '`<td>` cells using the `[headers]` attribute only refer to other cells in the same ' +
       'table may improve the experience for screen reader users. ' +
-      '[Learn more](https://dequeuniversity.com/rules/axe/4.4/td-headers-attr).',
+      '[Learn more about the headers attribute](https://dequeuniversity.com/rules/axe/4.4/td-headers-attr).',
```

Comment:
> \`headers\`

Also, here and elsewhere: should we be doing
> \`[headers]\`

? Could be a followup. Also may need parser support. Also might be worth inventing something like 

> @ header (ignore space, cant avoid github parsing at symbol) 
> or attr@header

as shorthand.

Reply:
I did investigate the possibility of supporting nested squares in md links. We're currently using [regex](https://github.com/alexnj/lighthouse/blob/master/report/renderer/util.js#L258) to match links, and a clean way would be to nest like in PCRE but it's not supported in JavaScript regex. 

An okay work-around would be a single level of nesting (as [I tried here](https://github.com/alexnj/lighthouse/commit/70f07cd8927328ae7c3f42f9842cfbb8513827e9)). That said, this only complicates the screen reader experience, without real benefits; so I didn't pursue. Wdyt?

Reply:
I'm thinking it'll be cumbersome to author strings like that. would much rather do a post-processing / annotation shorthand thing